### PR TITLE
Update Cargo.toml for first release

### PR DESCRIPTION
**Pull request by steveklabnik**
Add in other important things!

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -3,6 +3,13 @@ name = "simple-server"
 version = "0.1.0"
 license = "MIT/Apache-2.0"
 authors = ["steveklabnik <steve@steveklabnik.com>", "ashleygwilliams <ashley666ashley@gmail.com>"]
+description = "A simple webserver."
```
Comment (ashleygwilliams) [inline Cargo.toml]: no period- no capitals?

File: Cargo.toml [hunk 2]
```diff
@@ -3,6 +3,13 @@ name = "simple-server"
...
+repository = "https://github.com/steveklabnik/simple-server"
+readme = "README.md"
+categories = ["web-programming"]
```
Comment (ashleygwilliams) [inline Cargo.toml]: "web", "webserver", "server"

Comment (steveklabnik) [reply Cargo.toml]: This is set by crates.io, not us.

Comment (ashleygwilliams) [reply Cargo.toml]: FINE

**Pull request merged**